charts/sophora-ugc/templates/ingress.yaml

```diff
@@ -0,0 +1,36 @@
+{{- $fullName := include "sophora-ugc.fullname" . -}}
```

Comment:
We typically allow the chart user to enable/disable the ingress by convention. See: https://github.com/subshell/helm-charts/blob/main/charts/sophora-seo-check/templates/ingress.yaml for example. The default for `ingress.enabled` should be `false`.